### PR TITLE
Fix Consultations tab bug

### LINKED ISSUE #216: Solve GUI Consultations Tab Bug
Enable information to be displayed when the Consultations tab is clicked

### LINKED ISSUE #220: Add tab switching for add and edit commands

### PR DESCRIPTION
**Pull request by ngzhenteng**
Remove consultation related information from Student
Enable tab switch for all commands
Fixes #220 
Fixes #216

### CODE REVIEW

File: src/main/java/seedu/address/logic/Logic.java [hunk 1]
```diff
@@ -42,6 +43,9 @@
     /** Returns an unmodifiable view of the filtered list of quests */
     ObservableList<Quest> getFilteredQuestList();
 
+    /** Returns an unmodifiable view of the filtered list of consultations */
+    ObservableList<Consultation> getFilteredConsultationsList();
```
Comment (wireseo) [inline src/main/java/seedu/address/logic/Logic.java]: was this the problem? I'm still confused as to why I had the error in the first place XD

File: src/main/java/seedu/address/logic/commands/add/AddConsultationCommand.java [hunk 2]
```diff
@@ -23,7 +24,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.addConsultation(toAddConsultation);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAddConsultation));
+        return new CommandResult(String.format(MESSAGE_SUCCESS, toAddConsultation), CommandTargetFeature.Consultations);
```
Comment (wireseo) [inline src/main/java/seedu/address/logic/commands/add/AddConsultationCommand.java]: Nice tagging feature! Love it!

File: src/main/java/seedu/address/logic/Logic.java (hunk 1 above)
Comment (ngzhenteng) [reply src/main/java/seedu/address/logic/Logic.java]: Yeah this was the problem, i wrote consultations and you wrote consultation haha

File: src/main/java/seedu/address/logic/commands/add/AddConsultationCommand.java (hunk 2 above)
Comment (ngzhenteng) [reply src/main/java/seedu/address/logic/commands/add/AddConsultationCommand.java]: Thank you!

**Comment by wireseo** [comment]
LGTM! Again, thank you so much for sorting out the GUI bug. :)

File: src/main/java/seedu/address/logic/commands/add/AddCommand.java [hunk 3]
```diff
@@ -120,7 +121,7 @@ public CommandResult execute(Model model) throws CommandException {
             }
 
             model.addPerson(toAddStudent);
-            return new CommandResult(String.format(MESSAGE_SUCCESS, toAddStudent));
+            return new CommandResult(String.format(MESSAGE_SUCCESS, toAddStudent), CommandTargetFeature.Students);
```
Comment (Asuraxsoul) [inline src/main/java/seedu/address/logic/commands/add/AddCommand.java:124]: Thanks for adding feature for navigating back to the corresponding tabs

**Review by Asuraxsoul — COMMENTED** [review]
LGTM

**Pull request merged**